packages/unit-vitest/src/templates/base/src/components/___examples__/ExampleComponent.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
I'd avoid "__examples__" nesting, as we never used this convention with other AEs
Either we move these into test/vitest folder, or I'd squash this up one level changing components name if there's a conflict with Cypress tests

I'd go with the former, but I'm fine either way